test/api/api_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+ *    Copyright 2018 The Service Manager Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package api
+
+import (
+	"context"
+
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/Peripli/service-manager/bootstrap"
+	"github.com/Peripli/service-manager/env"
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+
+	"github.com/gavv/httpexpect"
```

Comment:
this is not listed as a dependency

Reply:
Still not listed in Gopkg.tolm see it

Reply:
It doesn't need to be in Gopkg.toml. It is in the lock file. Most of the dependencies are only in the lock file

Reply:
This way, the last commit from the master branch will be taken. I'd rather not have `branch = "master"`, but exact revision number for a guaranteed reproducible build.
Sure, this will work, but it's not good practice